libbeat/docs/loggingconfig.asciidoc

```diff
@@ -169,6 +169,15 @@ unix epoch. Defaults to disabled.
 
 When true, logs messages in JSON format. The default is false.
 
+[float]
+==== `logging.files.redirect_stderr` experimental[]
+
+When true, diagnostic messages printed to {beatname_uc}'s standard error output
```

Comment:
This only works with the file output, right? If so I'd add a note about that just so that no one expects it to work with syslog or event_log outputs.